integrations/extensions/starter-kits/metaweather/readme.md

```diff
@@ -14,11 +14,10 @@ The API reference for MetaWeather is at [https://www.metaweather.com/api/](https
 
```

Comment:
mybe I missed it,
where is there a description/comparison of advanced/basic for `metaweather`

Reply:
Working on it, it's supposed to be in a separate PR.. but maybe ill throw it all in here 😄